docker/Dockerfile

```diff
@@ -0,0 +1,192 @@
+# syntax=docker/dockerfile:experimental
+ARG USER=garage-user
+ARG UID=999
+ARG PARENT_IMAGE=ubuntu:18.04
+
+FROM $PARENT_IMAGE AS garage-18.04
```

Comment:
these should all be in the `rlworkgroup/` namespace, for when they get uploaded to Docker Hub